src/rgw/rgw_group.h

```diff
@@ -0,0 +1,132 @@
+#ifndef CEPH_RGW_GROUP_H
+#define CEPH_RGW_GROUP_H
```

Comment:
i'm just nitpicking over the coding style requirement for [self-contained headers](https://google.github.io/styleguide/cppguide.html#Self_contained_Headers) here

i think most of our sources get these dependencies through rgw_common.h. you can either include that, or express the fine-grained requirements (my preference)